src/character.h

```diff
@@ -290,9 +291,34 @@ class Character : public Creature
         Character(Character &&) = default;
         Character &operator=(const Character &) = default;
         Character &operator=(Character &&) = default;
-        
+        struct trait_data : public JsonSerializer, public JsonDeserializer {
+            /** Key to select the mutation in the UI. */
+            char key = ' ';
```

Comment:
So you want a separate map for the char key to mutation mapping? Should that be done for bionics as well?

It seems a bit pointless and error prone, one has to keep both maps synchronized all the time.